zcutil/make-release.py

```diff
@@ -123,17 +131,41 @@ def verify_releaseprev_tag(releaseprev):
         )
 
 
+@phase('Checking version.')
+def verify_version(release, releaseprev, hotfix):
+    if not hotfix:
+        return
+
+    expected = Version(
+        releaseprev.major,
+        releaseprev.minor,
+        releaseprev.patch,
+        releaseprev.betarc,
+        releaseprev.hotfix + 1 if releaseprev.hotfix else 1,
```

Comment:
Do hotfixes differ in their [CLIENT_VERSION](https://github.com/zcash/zcash/blob/master/src/clientversion.h#L55-L59) number? (I think they should.)

Reply:
Yes they will; the `releaseprev.hotfix` requirements above correspond to incrementing `CLIENT_VERSION_BUILD`, which is equivalent to incrementing `CLIENT_VERSION`.